### DISCUSSION TITLE
Does Sandboxie guard against BSODs?

### DISCUSSION THREAD

**Discussion by bastik-1001** [sandboxie-plus/Sandboxie#2976] [Q&A Feedback] [answered]
Will have sandboxing a program that can be triggered to cause a BSOD have any positive effect? Like it just crashing.

A virtual machine virtualizes to a greater extent, for example with an abstraction of the graphical interface, where it can help to migate these issues.

**Comment by DavidXanatos** [comment, marked as answer]
It depends on how the program triggers the BSOD, through a bug in windows or in a 3rd party driver.
If a bug in windows, than when we know exactly how, we could add a targeted mitigation.
If a bug in a 3rd party driver, using a security gardened box will block access to that driver hance no BSOD.

**Reply by bastik-1001** [reply]
Good to know that it can be helpful.
